app/jobs/fetch_track_update_job.rb

```diff
@@ -0,0 +1,26 @@
+class FetchTrackUpdateJob < ApplicationJob
+  attr_accessor :track_update_fetch_id
+
+  before_perform do |job|
+    track_update = TrackUpdate.find(job.arguments.first)
+
+    track_update_fetch = TrackUpdateFetch.create!(
+      track_update: track_update,
+      host: Socket.gethostname
```

Comment:
We need to think about Socket.gethostname in an ec2 context, but let's defer that until we've sorted everything else out. There are a couple of options here which would be fun to discuss separately.

Reply:
I think we can talk about this now. I've addressed your comments and I've got it working locally. This is the last bit we need to work on before we can deploy.

Reply:
so there's no harm using the hostname, but it'll change on reboot (unless we pin internal addresses). Alternatively, we could lookup the public IP from the EC2 api (static and straightforward), or we could get chef to write a file somewhere that configures a server slug.

Reply:
Would it matter if the hostname changed on reboot? I'm assuming so, but maybe not.

Reply:
I think it matters because we need to know whether the host has fetched the server or not. Does the public IP change only when we use new instance? It could also be a problem when the machine's IP changes while the instance doesn't.

Reply:
We do have fairly static IPs, but lets decouple this from the network identity, because as you hint, it's problematic. I'll adjust chef to write a slug into `/etc/exercism/server_identity` so if you just read from that file (and chomp) then we're good I think.